.vscode/launch.json

```diff
@@ -4,6 +4,14 @@
 // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
 {
   "version": "0.2.0",
+  "inputs": [
+    {
+      "id": "wingSource",
+      "type": "promptString",
+      "description": "Wing source path",
+      "default": "${workspaceFolder}/temp/tst.w"
```

Comment:
```suggestion
      "default": "${workspaceFolder}/examples/tests/valid/hello.w"
```

Reply:
you can take a look at that: 
https://github.com/winglang/wing/blob/ff09903def9e7bc3ddf0e7f1db2abbf8f9ff0dfb/.vscode/launch.json#L33
this runs the current file opened in the ide (and simpler to move between files without editing `hello.w`)

Reply:
@tsuf239 - anyway making it work using your example wasn't trivial, so I left it for another time